.github/workflows/build.yml

```diff
@@ -0,0 +1,242 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ dev ]
+
+jobs:
+
+
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        lua_ver: [51, 53]
+        numbers: ['float']
+        include:
+          - lua_ver: 51
+            numbers: 'integral'  
+          - lua_ver: 53
+            numbers: '64bit'  
+    runs-on: ubuntu-latest
```

Comment:
1. "latest" here seems to imply v18.04. On Travis, however, we use Xenial/16.04 as [we don't specify `dist: something`](https://github.com/nodemcu/nodemcu-firmware/blob/release/.travis.yml) and [Xenial is the default.](https://docs.travis-ci.com/user/reference/overview/). Is the 16->18 update intentional?
2. IMO we should pin the version rather then rely on whatever GitHub uses as "latest".

Reply:
Pinned. In docker we also use Xenial.